docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,149 @@
+it provides some additional convenience of not having to manually configure a PAM
+script to create and delete users.
+
+It also has benefits over manually creating users as it allows for separate host
+users for each member of an organization. This provides more fine-grained control
+of permissions on a given host.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster.
+- The following utilities should be available in the PATH for the Teleport SSH Service,
+since it must execute these commands in order to create transient users:
+  - `useradd`
+  - `userdel`
+  - `usermod`
+  - `groupadd`
+  - `getent`
+  - `visudo`
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Configure a role
+
+First, create a role with `create_host_user` set to `true`. This will allow users
+with this role to have transient host users created at login time.
+
+The following role specification will allow users to log in as `nginxrestarter` on
+any matching Node. The host user will be created and added to the groups listed in
+`host_groups`. They will also be given permission to restart the Nginx service as
+root.
+
+Save the below file as `auto-users.yaml`
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: auto-users
+spec:
+  options:
+    # allow automatic creation of users.
+    create_host_user: true
+  allow:
+    logins: [ "nginxrestarter" ]
+    # list of host groups the created user will be added to. Any that dont already exist are created.
+    host_groups: [ubuntu, nginx, other]
+    # list of entries to include in a temporary sudoers file created in /etc/sudoers.d
+    host_sudoers: [
+       # This line will allow the `nginxrestarter` user to run `systemctl restart nginx.service` as
+       # root without requiring a password.
+       #
+       # sudoers file reference documentation: https://www.sudo.ws/docs/man/1.8.17/sudoers.man/
+       "{{internal.logins}} ALL = (root) NOPASSWD: /usr/bin/systemctl restart nginx.service"
+    ]
+    node_labels:
+      'env': 'devel'
+```
+Create the role:
+```code
+$ tctl create -f auto-users.yaml
+# role 'auto-users' has been created
+```
+
+Each value of the `logins` field must conform to the username requirements
+of the Linux distribution being used. See [User/Group Name Syntax](https://systemd.io/USER_NAMES/) for requirements in common distributions.
+
+<Admonition type="warning">
+
+When a Teleport user accesses a Node, Teleport Server Access checks each of the
+user's roles that match the Node. If at least one role matches the Node but does not
+include `create_host_user: true`, automatic user creation will be disabled. Roles that
+do not match the Node will not be checked.
+
+</Admonition>
+
+If a role sets `host_sudoers` to '*' in the role deny section, the user will have all
```

Comment:
It would also be good to have another sentence re: how `host_sudoers` works in `deny` rules. What if the user adds a value other than `"*"`? So far, we only know what this one specific value does.